### DISCUSSION TITLE
scipy

### DISCUSSION THREAD

**Discussion by zolizolizoli** [flet-dev/flet#3391] [Packages] [answered]
https://pypi.org/project/scipy/

**Comment by abussinan** [comment]
Any progress for this or ETA?

**Comment by ndonkoHenri** [comment, marked as answer]
`scipy` 1.8.0 is now [available](https://pypi.flet.dev/scipy) for iOS and Android! It's a Fortran-free build, so the only piece left out is the deprecated `scipy.odr` — everything else should work normally. 

Give it a try and let know.
